app/services/proxy_status/base.rb

```diff
@@ -18,6 +18,15 @@ def self.humanized_name
       'Base'
     end
 
+    #Allow dumping status objects to the cache without dumping the api endpoint - so SSL certs won't break cache
```

Comment:
LGTM except the reasoning here, can you reword it to something more clear (SSL cert is too broad term). I'd go with

    # avoid marshalling @api

or similar.